### PR TITLE
Migrations

### LINKED ISSUE #24: A User can edit their profile
As a user,
When I visit /user/profile
and click on the `edit profile` button, 
I am taken to the update form where I see fields for:

- [x] `image`, `name`, `email`, `bio`, `street1`, `street2`, `city`,  `zip` 
     - [x] all is required, except `image` and `street2`
     
- [x] When I click `Update Profile`, I am redirected to my profile and see the updated information

**Potential Mutation**
```
// User update
export const userUpdate = {
  type: UserType,
  args: {
    id: {
      name: 'id',
      type: GraphQLInt
    },

    image: {
      name: 'image',
      type: GraphQLString
    },

    name: {
      name: 'name',
      type: GraphQLString
    },

    email: {
      name: 'email',
      type: GraphQLString
    },

    bio: {
      name: 'bio',
      type: GraphQLString
    },

    role: {
      name: 'role',
      type: GraphQLInt
    },

    shippingAddress: {
      name: 'shippingAddress',
      type: GraphQLString
    }
  },
  resolve: update
}
```

### PR DESCRIPTION
**Pull request by kevxo**
## Motivation and Context

closes #24 

## How Has This Been Tested?

No tests at the moment.

## Types of changes

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [] My code follows the code style of this project.
- [] I have updated the README to reflect any changes.

### CODE REVIEW

File: code/api/src/seeders/1-user.js [hunk 1]
```diff
@@ -20,6 +26,9 @@ module.exports = {
         email: 'user@crate.com',
         password: bcrypt.hashSync('123456', config.saltRounds),
         role: params.user.roles.user,
+        image: 'https://pbs.twimg.com/profile_images/1237550450/mstom_400x400.jpg',
+        bio: 'I am Tom from MySpace and I\'ve got style',
+        shippingAddress: JSON.stringify(jsonAddress),
```
Comment (carson-jardine) [inline code/api/src/seeders/1-user.js:31]: This makes sooo much sense. Looking forward to walking through this later!! LGTMMMMM

**Review by carson-jardine — CHANGES_REQUESTED** [review]
Nice work, Kevin! It looks good to be merged after we rebase it

**Review by ADavidson02 — APPROVED** [review]
looks good

File: code/api/src/seeders/1-user.js (hunk 1 above)
Comment (jaypeasee) [reply code/api/src/seeders/1-user.js:31]: Nice, Tom from Myspace

**Comment by jaypeasee** [comment]
Excited to try to hit this endpoint!

**Review by jaypeasee — APPROVED** [review]

**Pull request merged**